### DISCUSSION TITLE
Using Alpine Headless Listbox within a Livewire environment

### DISCUSSION THREAD

**Discussion by robdekort** [alpinejs/alpine#4398] [1. Help]
Hello there. I'm trying to use the Headless Listbox component as per this example: https://alpinejs.dev/component/headless-listbox

However, I'm working in a Livewire environment so my bundle looks like this:

```js
import { Livewire, Alpine } from '../../vendor/livewire/livewire/dist/livewire.esm'
import precognition from 'laravel-precognition-alpine'
import ui from '@alpinejs/ui'

Alpine.plugin(precognition, ui)
Livewire.start()
```

Livewire and basic components work just fine. However with the listbox I get the following errors: `ReferenceError: Can't find variable: $listboxOption`

<img width="742" alt="Screenshot 2024-10-16 at 15 51 16" src="https://github.com/user-attachments/assets/20e05902-ee26-4cc3-8ed4-f261ebb5ab24">

I'm wondering how to load in the Alpine Components properly to prevent this from happening. If anyone has any thoughts, much appreciated.

**Comment by ekwoka** [comment]
To me this seems like none of the plugins are activating.

Which might be more appropriate on the Livewire repo.

Are you possibly not removing the `livewire scripts` tag or whatever it is?

**Reply by robdekort** [reply]
Thanks for your answer, much appreciated. Nope, as on the same page livewire itself works.

**Reply by robdekort** [reply]
And so do other custom alpine components.

**Reply by ekwoka** [reply]
So you are or are not removing livewire scripts?

Don't you need to remove livewire scripts to separately activate alpine?

It's a livewire issue though.

**Reply by robdekort** [reply]
I do the `scriptsConfig` thing if that's what you mean?

**Reply by robdekort** [reply]
I'll move it to Livewire though. Thanks.
